### DISCUSSION TITLE
Problem with xRDP remote GUI desktop in Security Onion and Remmina in (K)Ubuntu

### DISCUSSION THREAD

**Discussion by Maba79** [Security-Onion-Solutions/securityonion#2517] [Unsupported Versions] [answered]
I'm trying to set up an internal remote GUI desktop session into my SO lab with xRDP and my (K)Ubuntu laptop through my own LAN. I've managed to set up access via Windows RDP client (Bitvise SSH) and it works fine, but I'm struggling with my (K)Ubuntu and Remmina client (installed version 1.4.7, using RDP).

I have already configured SSH and I have also allowed so-allow 'analyst' FW rule from SO UFW for web portal access to Kibana, Squert and Cyberchef. They all work fine. However, I wish to practice triage with Squil and do also some other stuff for which I prefer remote GUI  access.

I have xRDP server installed in my SO and it runs fine. SO has also the latest updates. I've also installed xfce4 desktop into SO according to these [instructions](https://secatlas.wordpress.com/2015/03/09/security-onion-and-xrdp/) and it works great for all Windows RDP  sessions. However, whenever I try to access it via (K)Ubuntu's Remmina client (via RDP protocol), connection seems to establish fine, but the screen stays black and nothing happens. There seems to be some small graphical errors/glitches displayed every now then. I've also tried enabling and disabling various Remmina settings, e.g. edited Remmina's RDP connection -> Advanced-Tab -> scrolled down to bottom and checked "Relax Order Checks" and "Glyph Cache" according to these [instructions](https://gitlab.com/Remmina/Remmina/-/issues/1699). I've also tried Remmina's different graphic/display quality settings, but nothing seems to work.

I have opened ports 3389, 5900 and 3350 from my SO UFW.  Now I've ran out of all ideas, so any help would be greatly appreciated. Thanks.

**Comment by dougburks** [comment, marked as answer]
Since you mention Squert and Sguil, I'll assume you're using the old Security Onion 16.04, which reaches End Of Life in less than 4 months:
https://blog.securityonion.net/2020/12/4-month-eol-notice-for-security-onion.html

This Discussions page is for the new Security Onion 2. Your best bet is to install Security Onion 2. All analyst interfaces are web based, so you won't need to worry about xrdp or Remmina.

**Reply by rock-kobs** [reply]
since this is related to xrdp, how can we open 3389 for xrdp or any other port to customise our SO solutions. the documents on this is not very clear because when i do netstat my customise ports shows its listening but still cannot connect. am using the latest update at the time of writing this question.

**Reply by dougburks** [reply]
@rock-kobs This is an old discussion about the old Security Onion 16.04 which is now past End Of Life. In the new Security Onion 2, all analyst interfaces are web based so you shouldn't need to use xrdp. If you are using the new Security Onion 2 and have further questions or problems, please feel free to start a separate discussion. Thanks!

**Reply by rock-kobs** [reply]
thanks @dougburks, I will do that, just some quick addition, some of us loves to try different scenarios to see how far we can customise this amazing tool. one experiment I tried was using ubuntu with onion and using as normal OS had performance issues but when I used Dell R710 with 80g ram 24 cores it worked well. anyway thanks for responding. hope you have a good day.
